src/webhook/handlers/choosingArticle.js

```diff
@@ -365,9 +363,11 @@ export default async function choosingArticle(params) {
         }
       }
     `({ id: selectedArticleId }, { userId });
+
+    state = '__INIT__';
```

Comment:
I do have a question here.

It seems that for most event handlers, we no longer return a `state` anymore. We choose to detect what "state" the user should go to in `handleInput` instead.

If this is the case, should we still mutate `state` here?

Reply:
No. I return `state: CHOOSING_REPLY` at line 127 is because `isSkipUser: true`, which makes `handleInput` continue `choosingReply`. 

Maybe we can consider calling `choosingReply` directly or use a callback function at line 127 and remove `isSkipUser` do-while loop in `handleInput`. 
Note: there's another `isSkipUser: true` in `initState`.